srcpkgs/swappy/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'swappy'
+pkgname=swappy
+version=1.1.0
+revision=1
+build_style=meson
+hostmakedepends="pkg-config scdoc glib-devel"
+makedepends="cairo-devel glib-devel gtk+3-devel libnotify-devel pango-devel"
+depends="wl-clipboard font-awesome5"
```

Comment:
A snapping tool depends on a specific font?
Something was wrong?

Reply:
I agree, it's quite strange but this font is what they use for button icons. Without the font installed the icons show up as unknown characters.

Reply:
maybe it can be patched out?